colossus/src/main/scala/colossus/core/WorkerManager.scala

```diff
@@ -219,6 +219,22 @@ extends Actor with ActorLogging with Stash {
   }
 }
 
+private[colossus] trait WorkerFactory {
+  def createWorker(id: Int, ioSystem: IOSystem, context: ActorContext): ActorRef
+}
+
+private[colossus] object DefaultWorkerFactory extends WorkerFactory{
```

Comment:
zee original code which was in WorkerManager for creating Workers.  Only moved, not changed.